odl/test/solvers/functional/functional_test.py

```diff
@@ -131,7 +137,9 @@ def test_derivative(functional):
     the inner product of the gradient and the direction, if the gradient is
     defined.
     """
-    if isinstance(functional, odl.solvers.functional.IndicatorLpUnitBall):
+    if isinstance(functional, (odl.solvers.functional.IndicatorLpUnitBall,
```

Comment:
Create a global, e.g. "FUNCTIONALS_WITHOUT_DERIVATIVE" and use it instead of repeating yourself. That also fixes the comment in the line below that is now lacking